core/templates/pages/exploration-player-page/services/player-position.service.ts

```diff
@@ -58,6 +64,10 @@ export class PlayerPositionService {
     }
   }
 
+  /**
+   * Set the index of the displayed card.
+   * @param {object} index - A number which represents the index of card.
```

Comment:
Should be a number and not an object, right?
```suggestion
   * @param {number} index - The new index of the card.
```